src/tools/faq.md

```diff
@@ -45,11 +45,9 @@ hoping to address the root cause but it will take a while to get there.
 
 #### Q. What are all the "packages" directories for?
```

Comment:
If we're removing references to the `packages/` directory elsewhere, I think we can safely ditch this question as well. It won't be "frequently asked" anymore.